test/cpp/qps/client_async.cc

```diff
@@ -130,39 +130,26 @@ class ClientRpcContextUnaryImpl : public ClientRpcContext {
       response_reader_;
 };
 
-class AsyncUnaryClient GRPC_FINAL : public Client {
+class AsyncClient : public Client {
  public:
-  explicit AsyncUnaryClient(const ClientConfig& config) : Client(config) {
+  explicit AsyncClient(const ClientConfig& config,
+                       void (*setup_ctx)(CompletionQueue*, TestService::Stub*,
+                                         const SimpleRequest&)) :
+      Client(config) {
     for (int i = 0; i < config.async_client_threads(); i++) {
       cli_cqs_.emplace_back(new CompletionQueue);
     }
-
-    auto check_done = [](grpc::Status s, SimpleResponse* response) {};
-
     int t = 0;
     for (int i = 0; i < config.outstanding_rpcs_per_channel(); i++) {
       for (auto channel = channels_.begin(); channel != channels_.end();
```

Comment:
And just to followup, the QPS test is currently above-the-law with regard to that issue, but I don't want to reintroduce removed features like foreach just in case we do decide to bring it into full compliance with the rest of the codebase.